lib/read.js

```diff
@@ -102,13 +99,20 @@ class ReadOpenTSDB extends AdapterRead {
 
     addTimeLimitsToClient(from, to) {
         if (from) {
-            this.client.start(from.milliseconds());
-            this.logger.debug('start time', JSON.stringify(from));
+            let start = this.getAbsoluteTime(from.clone());
+            this.client.start(start);
+            this.logger.debug('start time', start);
         }
-
-        let end = to || new Date();
-        this.client.end(end.milliseconds());
-        this.logger.debug('end time', JSON.stringify(end));
+        
+        let end = to || new JuttleMoment({rawDate: new Date()});
+        end = this.getAbsoluteTime(end.clone());
+        this.client.end(end);
+        this.logger.debug('end time', end);
+    }
+    
+    getAbsoluteTime(juttleMoment) {
+        return juttleMoment.moment.format()
```

Comment:
What about passing `format` argument directly? `JuttleMoment.format(juttleMoment, 'YYYY/MM/...')`. Moment.js format is [hairy and non-unixish](http://momentjs.com/docs/#/displaying/format/), but I think it is better to write it out,so if you read the code later, you know which format is actually used :)